internal/builtinchecks/yamls/dangling-deployment.yaml

```diff
@@ -0,0 +1,7 @@
+name: "dangling-deployment"
+description: "Alert on deployments that don't have any selector matching a label in pod template"
+remediation: "Make sure your deployment's selector correctly matches a label in pod template."
+scope:
+  objectKinds:
+    - Deployment
```

Comment:
This needs to be `DeploymentLike` (this was causing the error you were seeing, the object kind here has to match one of the kinds registered in `internal/objectkinds`).

Reply:
Done